src/api/app/models/event/create_project.rb

```diff
@@ -19,7 +19,7 @@ def metric_tags
     end
 
     def metric_fields
-      { value: 1 }
+      { project: payload['project'], count: Project.count }
```

Comment:
Why do you replace it? Why don't you send both?

Reply:
I would put the project as tags. Tags are supposed to be finite but it doesn't matter if you have a large amount of tags. Tags are indexed and very performant. Then you can do queries like ```where project = Factory```